funnel/templates/schedule_edit.html.jinja2

```diff
@@ -12,6 +12,11 @@
 {% endblock %}
 
 {% block top_title %}
+  {% if project.schedule_state.PUBLISHED and not project.schedule_start_at %}
+    <div class="alert alert--warning">
+      <p class="alert__text">Please add session to the schedule for the project to be listed on the profile and homepage.</p>
+    </div>
+  {% endif %}
```

Comment:
This warning may not work as the page does not reload when individual sessions are added or removed.

Reply:
Do I add this check while editing session ?

Reply:
<img width="450" alt="Screen Shot 2020-05-11 at 3 21 54 PM" src="https://user-images.githubusercontent.com/11267213/81548672-43819080-939b-11ea-9724-c2e73b1dd66c.png">

Reply:
This works though it's easy to miss.